src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 200px;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  border: 1px solid $background-color;
+  border-radius: 5px;
+
+  &:hover {
+    transition: 0.3s;
```

Comment:
The `transition` property should be set on the main class, not on the `:hover` pseudo-class. This ensures that the transition effect is applied both when the element is hovered over and when the hover is removed.